src/Jobs/Invite.php

```diff
@@ -88,18 +90,44 @@ public function handle()
      */
     private function inviteUserIntoGuild()
     {
-        $new_nickname = optional($this->discord_user->group->main_character)->name;
 
-        $user = app('discord')->guild->addGuildMember([
-            'user.id'      => $this->discord_user->discord_id,
-            'guild.id'     => intval(setting('warlof.discord-connector.credentials.guild_id', true)),
-            'nick'         => ! is_null($new_nickname) ? $new_nickname : $this->discord_user->nick,
-            'access_token' => $this->getAccessToken(),
+        $corp_id = $this->discord_user->group->main_character->corporation()->corporation_id;
```

Comment:
This can be simplified a bit.

`corporation` is a relation link to `CorporationInfo` - so if you're using it, you can directly set `$corp = $discord_user->group->main_character->corporation;`.

In all case, you have to ensure you got a `CorporationInfo` instance, otherwise you'll get exception later in the code.